### ISSUE TITLE
How did you train models?

### ISSUE THREAD

**Issue by nomurakeiya** [TencentYoutuResearch/CrowdCounting-SASNet#1]
thanks for good code!

I want to know how did you train models(SHHA.pth,SHHB.pth)?

**Comment by WYG-2021** [comment]
yeah，i want to know too.
can you share your code of the trainning part?

**Comment by feixuedudiao** [comment]
yeah, i want too.

**Comment by AaronDavis2020** [comment]
yeah, i want too. thanks.

**Comment by icoicqico** [comment]
Yes, I want the training script too. thanks a lot.

**Comment by netanelbi** [comment]
Can i have the Training script as well ?
thanks a lot :)

**Comment by evalmodel** [comment]
Hi, can i have the training script as well ?
Thanks :)

**Comment by chocokassy** [comment]
hi, I want train code too, Thanks!

**Comment by thomassoh** [comment]
Could you provide the training script if it is not forbidden by some of your policies?
Thanks!

**Comment by alreem-almuh** [comment]
can you share your code for training part?

**Comment by 18339508629** [comment]
Hello, I'm honored to read your paper. Can you refer to your train.py and PRA loss? If you can, please send it to 1125997761@qq.com . Thank you very much!

**Comment by twiQ1** [comment]
那你们最终拿到代码了吗？

**Comment by khachuy1904** [comment]
Hello, could I please have the training code. If possible, please send it to this email address: nguyenkhachuy1904@gmail.com
.
Thanks!
